tests/web/test_assets.py

```diff
@@ -18,7 +18,7 @@ def test_get_assets_with_missing_text_search(client):
 
 
 @pytest.mark.genesis
-def test_get_assets(client, b):
```

Comment:
This test should be removed in favour of [this](https://github.com/bigchaindb/bigchaindb/pull/2338/files#diff-5887167ed95ccb195dfe41eb9fd2b03cR91)